incubator/nfs-pv-provider/values.yaml

```diff
@@ -0,0 +1,14 @@
+image:
+  repository: quay.io/kingdonb/nfs-data
+  tag: master
+  pullPolicy: IfNotPresent
+
+storage:
+  name: nfs-pv-provisioning-demo
+  demo_label: nfs-pv-provisioning
+  capacity: 50Gi
+  storageClass: default
+
+deployment:
+  api_version: apps/v1beta2
```

Comment:
Camel case please, but why would you make this configurable? Why not just use `apps/v1beta2`?

Reply:
At this point I guess it does not really matter, as I've learned recently there is not a release of K8s out yet that will balk at v1beta2... so OK, I'll set it to be fixed as `apps/v1beta2` api.